<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmSelectGear</class>
 <widget class="QWidget" name="CArmSelectGear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>135</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>135</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{background-color:rgba(14,14,14,1);
border:1px solid #4A525A;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="oneGear">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="twoGear">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="threeGear">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
